Dfe.Data.SearchPrototype.Data/ManageData.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Dfe.Data.SearchPrototype.Data.Configuration;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dfe.Data.SearchPrototype.Data;
+
+public class ManageData
+{
+    private const int BatchSize = 1000;
+    public static async Task ExtractAndUploadData(AzureSearchServiceDetails searchDetails, HostApplicationBuilder builder)
```

Comment:
I would pass in the path as a string and do the extraction of the settings withing the program.cs file, feels like the Data manager class shouldn't care about the HostAppBuilder but rather receiving a file path it can try and apply :)

Reply:
Just thinking out loud here, wondering if we're happy to let this explode in it's current state if we have a dodgy file path or apply dodgy settings or whether we could wrap a try/catch and try and offer some feedback on app failure, not too precious at this stage though #FoodForThought